src/Core/SettingsChangesHistory.cpp

```diff
@@ -104,7 +104,8 @@ const VersionToSettingsChangesMap & getSettingsChangesHistory()
             {"evaluation_time", 0, 0, "New experimental setting"},
             {"output_format_parquet_date_as_uint16", false, false, "Added a compatibility setting for a minor compatibility-breaking change introduced back in 24.12."},
             {"enable_lightweight_update", false, true, "Lightweight updates were moved to Beta. Added an alias for setting 'allow_experimental_lightweight_update'."},
-            {"allow_experimental_lightweight_update", false, true, "Lightweight updates were moved to Beta."}
+            {"allow_experimental_lightweight_update", false, true, "Lightweight updates were moved to Beta."},
+            {"return_ref_table_for_alias_storage", true, true, "New setting"},
```

Comment:
```suggestion
            {"return_ref_table_for_alias_storage", false, true, "New setting"},
```

Reply:
add please move this line to 25.9, because the branch of 25.8 has been created.
How about rename this setting as `always_use_table_for_alias_storage`?